src/components/ContactForm/index.js

```diff
@@ -0,0 +1,170 @@
+  const [showErrors, setShowErrors] = useState(false);
+
+  const validEmail = value => {
+    return value && /^[^\s@]+@[^\s@]+\.[^\s@]+$/.test(value);
+  };
+
+  const onChangeName = event => {
+    handleChange(event);
+  };
+  const onChangeEmail = event => {
+    handleChange(event);
+  };
+  const onChangeMessage = event => {
+    handleChange(event);
+  };
+
+  const renderInput = (title, id, error, onChange) => {
+    const styles = classNames("input", {
+      border: error,
+    });
+
+    return (
+      <>
+        <Typography weight="medium" variant="small-body" color="oxford-blue">
+          {title}
+        </Typography>
+        <input
+          onChange={onChange}
+          type="text"
+          name={id}
+          id={id}
+          data-name={title}
+          styleName={styles}
+        />
+      </>
+    );
+  };
+
+  const renderName = () => {
+    const error = !formValues.name && showErrors;
+    const errorStyles = classNames("error", {
+      visible: error,
+    });
+
+    return (
+      <label htmlFor="name" styleName="label">
+        {renderInput("Name", "name", error, onChangeName)}
+        <div styleName={errorStyles}>
+          <Typography variant="xsmall-body" color="klein-blue">
+            Please enter your name
+          </Typography>
+        </div>
+      </label>
+    );
+  };
+
+  const renderEmail = () => {
+    const error = !validEmail(formValues.email) && showErrors;
+    const errorStyles = classNames("error", {
+      visible: error,
+    });
+
+    return (
+      <label htmlFor="email" styleName="label">
+        {renderInput("Email", "email", error, onChangeEmail)}
+        <div styleName={errorStyles}>
+          <Typography variant="xsmall-body" color="klein-blue">
+            Please enter a valid email address
+          </Typography>
+        </div>
+      </label>
+    );
+  };
+
+  const renderMessage = () => {
+    const error = !formValues.message && showErrors;
+    const styles = classNames("textarea", {
+      border: error,
+    });
+    const errorStyles = classNames("error", {
+      visible: error,
+    });
+
+    return (
+      <label htmlFor="message" styleName="message">
+        <Typography weight="medium" variant="small-body" color="oxford-blue">
+          Message
+        </Typography>
+        <textarea
+          onChange={onChangeMessage}
+          name="message"
+          id="message"
+          rows="5"
+          data-name="Message"
+          styleName={styles}
+        />
+        <div styleName={errorStyles}>
+          <Typography variant="xsmall-body" color="klein-blue">
+            Please write your message
+          </Typography>
+        </div>
+      </label>
+    );
+  };
+
+  const checkFormValidity = () => {
+    return (
+      validEmail(formValues.email) && formValues.message && formValues.name
+    );
+  };
+
+  const handleSubmit = event => {
+    const isFormValid = checkFormValidity();
```

Comment:
sorry for being picky but we don't need the variable, we can use directly `!checkFormValidity `, since the function job is clear